davidsymonds/David_Symonds.md

```diff
@@ -0,0 +1,37 @@
+# A Taxing Situation
+
+## How the UK poll tax lead to a career in software testing.
+
+Wikipedia describes The Community Charge commonly known as the poll tax as “a system of taxation introduced by Margaret Thatcher's government in replacement of domestic rates in Scotland from 1989, prior to its introduction in England and Wales from 1990. It provided for a single flat-rate per-capita tax on every adult, at a rate set by the local authority. The charge was replaced by Council Tax in 1993, two years after its abolition was announced.”  Its introduction lead to riots, punk rock songs, the fall of the one of the UKs most successful Prime Ministers and my move into Software Testing.
+
+I had accepted a job with British Telecom (BT) as a test manager on a poll tax system for 14 local authorities. At that time there were only two telecom companies in the UK. The largest BT was contracted to provide a system to package and print the poll tax bills, payment books and ancillary documents that local authorities sent out each year. The system also printed the individual Local Authorities coats of arms on the documents. This project had started in the previous October. It was running late and the documents had to be produced before the beginning of April. I and my six colleagues had to get the project back on track. To this end we worked the extended hours expected nowadays from a Start-up.  
```

Comment:
Does "This project had started in the previous October" work a bit more consicely as `This project started the previous October`?

Reply:
Yes. I would have written that but he was telling a story to me and I'd like to keep those archaic bits of grammar.